framework/src/mesh/MooseMesh.C

```diff
@@ -2495,6 +2495,21 @@ MooseMesh::effectiveSpatialDimension() const
   return 1;
 }
 
+unsigned int
+MooseMesh::getBlocksDimension(const std::vector<SubdomainName> & blocks) const
+{
+  unsigned short dim = 0;
+  for (const auto & subdomain_name : blocks)
```

Comment:
there's a range with subdomains that will condense these two loops

Reply:
You mean `active_local_subdomain_set_elements_begin`/`active_local_subdomain_set_elements_end`?

Reply:
Or I guess just `active_local_subdomain_set_elements_ptr_range`